### PR TITLE
Do some clean-up items for PR 406 that were missed

### PR DESCRIPTION
**Pull request by duglin**
Just minor tweaks

Signed-off-by: Doug Davis <dug@us.ibm.com>

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -212,12 +212,12 @@ within the same JSON object.
...
   scenarios where middleware is unable to interpret the `data` content. In the
   above example, the subscriber might only be interested in blobs with names
-  ending with '.jpg' or '.jpeg' and the subject attribute allows for 
+  ending with '.jpg' or '.jpeg' and the `subject` attribute allows for
   constructing a simple and efficient string-suffix filter for that subset of
   events.
 * Constraints:
   * OPTIONAL
-  * MUST be a non-empty string
+  * if present, MUST be a non-empty string
```
Comment (n3wscott) [inline spec.md]: `If`

File: spec.md [hunk 2]
```diff
@@ -212,12 +212,12 @@ within the same JSON object.
   the `data` payload) is particularly helpful in generic subscription filtering
   scenarios where middleware is unable to interpret the `data` content. In the
   above example, the subscriber might only be interested in blobs with names
-  ending with '.jpg' or '.jpeg' and the subject attribute allows for 
+  ending with '.jpg' or '.jpeg' and the `subject` attribute allows for
   constructing a simple and efficient string-suffix filter for that subset of
   events.
 * Constraints:
```
Comment (n3wscott) [inline spec.md]: All `*` to `-`

Comment (duglin) [reply spec.md]: think I got 'em all

File: spec.md (hunk 1 above)
Comment (duglin) [reply spec.md]: got it - thanks

**Comment by duglin** [comment]
@n3wscott PTAL!

**Comment by n3wscott** [comment]
LGTM

**Comment by duglin** [comment]
Approved on the 4/11 call

**Pull request merged**